deploy/olm-catalog/infinispan-operator.clusterserviceversion.yaml

```diff
@@ -57,7 +56,6 @@ metadata:
               "extraJvmOpts": "-Djava.property=me",
               "memory": "1Gi"
             },
```

Comment:
`,` needs to be removed as the block is now last element, otherwise it's going to result in invalid JSON